src/tools/ImageTransformer.cpp

```diff
@@ -0,0 +1,27 @@
+// Created by Maxandre Ogeret.
+// (c) 2022 University of Tartu - Autonomous Driving Lab.
+
+#include "tools/ImageTransformer.h"
+
+ImageTransformer::ImageTransformer(DriveworksApiWrapper* driveworksApiWrapper)
```

Comment:
Please add some comments what it does and why it is needed.

Reply:
Done.